core/src/main/java/com/novoda/accessibility/ActionsMenuAccessibilityDelegate.java

```diff
@@ -0,0 +1,39 @@
+package com.novoda.accessibility;
+
+import android.os.Bundle;
+import android.support.v4.view.AccessibilityDelegateCompat;
+import android.support.v4.view.accessibility.AccessibilityNodeInfoCompat;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+
+public class ActionsMenuAccessibilityDelegate extends AccessibilityDelegateCompat {
+
+    private final Menu menu;
+    private final MenuItem.OnMenuItemClickListener menuItemClickListener;
+
+    public ActionsMenuAccessibilityDelegate(Menu menu, MenuItem.OnMenuItemClickListener menuItemClickListener) {
+        this.menu = menu;
+        this.menuItemClickListener = menuItemClickListener;
+    }
+
+    @Override
+    public void onInitializeAccessibilityNodeInfo(View host, AccessibilityNodeInfoCompat info) {
+        super.onInitializeAccessibilityNodeInfo(host, info);
+        for (int i = 0; i < menu.size(); i++) {
+            MenuItem item = menu.getItem(i);
+            info.addAction(new AccessibilityNodeInfoCompat.AccessibilityActionCompat(item.getItemId(), item.getTitle()));
+        }
+    }
+
+    @Override
+    public boolean performAccessibilityAction(View host, int actionId, Bundle args) {
+        MenuItem item = menu.findItem(actionId);
+        if (item == null) {
+            return super.performAccessibilityAction(host, actionId, args);
+        } else {
+            menuItemClickListener.onMenuItemClick(item);
+            return true;
```

Comment:
same as above regarding the return value of `onMenuItemClick`. could delegate to super if false?

Reply:
yep makes sense 👍

Reply:
https://github.com/novoda/accessibilitools/pull/28#discussion_r178802623